front/SettingsForm.tsx

```diff
@@ -89,6 +91,7 @@ const SettingsForm = () => {
             <input
               className="form-control form-control-lg"
               type="email"
+              data-cy = "settings-email"
```

Comment:
Remove the space between 'data-cy' and '='. It should be written as 'data-cy="settings-email"'